mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+site_name: scp-machine
+
+extra_css:
+  - _assets/main.css
+extra_javascript:
+  - _assets/scg.min.js
+  - _assets/main.js
+
+nav:
+  - Home: index.md
+  - Quick start: quick_start.md
+  - Build instructions:
+      - Quick start: build/quick_start.md
+      - Build system: build/build_system.md
```

Comment:
```suggestion
  - Quick Start: quick_start.md
  - Build Instructions:
      - Quick Start For Developers: build/quick_start.md
      - Build System: build/build_system.md
```

Reply:
We don't use that style in the sc-machine